core/src/main/java/io/snyk/plugins/artifactory/scanner/ScannerModule.java

```diff
@@ -57,16 +57,29 @@ public void scanArtifact(@Nonnull RepoPath repoPath) {
     validateLicenseIssues(testResult, repoPath);
   }
 
-  protected Optional<PackageScanner> getScannerForPackageType(String path) {
+  protected PackageScanner getScannerForPackageType(String path) {
```

Comment:
Changed this to throw so that we can provide a reason why a scanner wasn't provided.